fiat-sql/src/main/kotlin/com/netflix/spinnaker/fiat/permissions/SqlPermissionsRepository.kt

```diff
@@ -110,6 +110,10 @@ class SqlPermissionsRepository(
         return this
     }
 
+    override fun putAllById(permissions: MutableMap<String, UserPermission>?) {
```

Comment:
Are you planning on mutating this map?  If not, may as we just be a `Map`.

Reply:
Good point. I'll change  that since I can't think of a reason to mutate the map there.